ui/app/styles/core/forms.scss

```diff
@@ -11,6 +11,13 @@ label {
   }
 }
 
+.masked-font {
+  font-family: obscure;
+  font-size: $size-medium;
+  letter-spacing: 2px;
+  color: $ui-gray-300;
```

Comment:
If this should never be another color, you could put `!important` so you don't have to overwrite it later. Up to you.